### ISSUE TITLE
[AVT], StackedBarChart/ LineChart, Contrast of foreground and background must be sufficient to meet WCAG AA requirements

### ISSUE THREAD

**Issue by PAnilReddy** [carbon-design-system/carbon-charts#635]
Charts version
"@carbon/charts": "^0.30.10",

Contrast of foreground and background must be sufficient to meet WCAG AA requirements violations are seen in the 2 charts we use StackedBarChart/ LineChart
12 violations in LineChart
11 violations in StackedBarChart

![image](https://user-images.githubusercontent.com/44162180/82661949-a5b77c80-9c4a-11ea-91f4-1797cc1bc011.png)

**Comment by theiliad** [comment]
seems like the tool is throwing violations for an invisible piece of text, closing the issue here
